packages/tests/src/tests/utils.ts

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2018-2021 TON DEV SOLUTIONS LTD.
+ *
+ * Licensed under the SOFTWARE EVALUATION License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at:
+ *
+ * http://www.ton.dev/licenses
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific TON DEV software governing permissions and
+ * limitations under the License.
+ */
+
+import { runner } from "../runner";
+import { test, expect } from "../jest";
+import { 
+    addressStringFormatAccountId,
+    addressStringFormatBase64,
+    addressStringFormatHex
+} from "@tonclient/core";
+
+test('utils: convert_address', async () => {
```

Comment:
I suggest to use double quotes in JS code.
Now we are using double quotes because of copy/paste of the code between languages. Major languages uses double quotes for strings so copy/paste is a slightly convenient.